packages/content-conformance/src/rules/ensure-valid-link-format.js

```diff
@@ -0,0 +1,207 @@
+const TEST_URL_ORIGIN = 'https://test.com'
+const LEARN_HOSTNAME = 'learn.hashicorp.com'
+const DEV_DOT_HOSTNAME = 'developer.hashicorp.com'
+
+const PRODUCT_SLUGS_TO_HOST_NAMES = {
+  boundary: 'boundaryproject.io',
+  consul: 'consul.io',
+  hcp: 'cloud.hashicorp.com',
+  nomad: 'nomadproject.io',
+  packer: 'packer.io',
+  terraform: 'terraform.io',
+  vagrant: 'vagrantup.com',
+  vault: 'vaultproject.io',
+  waypoint: 'waypointproject.io',
+}
+
+// TODO(brkalow): it would be good to have this come from a shared source
```

Comment:
+1 on this

Reply:
:nod: I'm thinking a shared package here might make sense, as we need this data in dev-portal too, and so having a single source of truth would be great to avoid drift.